.gitignore

```diff
@@ -1,8 +1,11 @@
 # Byte-compiled / optimized / DLL files
 __pycache__/
-*.py[cod]
 *$py.class
 
+# Setting up of PyCharm
+.idea/
+*.sqlite3
```

Comment:
I am not so sure anyone would keep and kind of sqlite3 file out of source control. Maybe it's better also to include that on gitignore global locally

Reply:
Ok! The same form of the first question I do it for the next PR!

Reply:
In fact you don't need to raise another PR. You can just fix those in your repo and once you push the fix, this PR will be fixed as well ;)